apollo-compiler/src/main/kotlin/com/apollostack/compiler/ir/GraphQlType.kt

```diff
@@ -0,0 +1,49 @@
+package com.apollostack.compiler.ir
+
+import com.squareup.javapoet.ClassName
+import com.squareup.javapoet.ParameterizedTypeName
+import com.squareup.javapoet.TypeName
+
+sealed class GraphQlType(val nullable: Boolean) {
+
+  class GraphQlString(nullable: Boolean = true) : GraphQlType(nullable)
+
+  class GraphQlId(nullable: Boolean = true) : GraphQlType(nullable)
+
+  class GraphQlInt(nullable: Boolean = true) : GraphQlType(nullable)
```

Comment:
sounds like you don't need to add a default value for the `nullable` argument, since it's only called from `resolveByName()` where you always pass the argument?
Also I wouldn't want to make them nullable by default, sounds like a dangerous decision.